src/java/org/apache/cassandra/config/Config.java

```diff
@@ -544,6 +544,8 @@ public static class SSTableConfig
     public volatile int tombstone_warn_threshold = 1000;
     public volatile int tombstone_failure_threshold = 100000;
 
+    public TombstonesMetricGranularity purgeable_tobmstones_metric_granularity = TombstonesMetricGranularity.row;
```

Comment:
@netudima why does not this have documented configuration property in `cassandra.yaml` and `cassandra_latest.yaml`? Is there some reason we are not exposing this property to a user to use so we have it just here, undocumented?

Reply:
@netudima there is also a typo in `tobmstones`.

Reply:
I would most probably also renamed this property. As of know, if you wake me up in the middle of the night and asked under what name the property was added I could not tell :) But I could tell you that it has something common _with tombstones_.

There are two tombstones properties just above starting on `tombstone` so I would do the same here. Maybe call it like `tombstone_purge_metric_granularity`.

Reply:
> why does not this have documented configuration property in cassandra.yaml and cassandra_latest.yaml?

Sorry,I just missed it, there was no an intentional reason. I will add it. Maybe we should think about an automatic test to check such consistency (as a part of a separate ticket, of course)

> there is also a typo in tobmstones.

good catch, thank you!

> tombstone_purge_metric_granularity

it sounds a bit strange for me because purgeable was an adjective here and now it looks more like a verb, so people may think that the metric is about a purging activity itself .. But I do not see a problem to change the name to tombstone_purge_metric_granularity if you think it is better.

Reply:
yeah please change it to that.